builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -536,7 +536,17 @@ def buildSlsApp(String runtime, String repo_name) {
       sh "mkdir -p $GOPATH/src"
       sh "rsync -a --exclude='.*' $GOPATH/" + repo_name + " $GOPATH/src"
       sh "cd $GOPATH/src/"+ repo_name+" && dep ensure"
-      sh "env GOOS=linux GOARCH=amd64 go build -o main $GOPATH/src/" + repo_name + "/main.go"
+      dirs = readDir("${env.WORKSPACE}/src/${repo_name}/functions")
+      def size = dirs.size()
+      if( size > 0) {
+        for(item in dirs) {
+          def functionpath = "$GOPATH/src/${repo_name}/functions/${item}"
+          sh "sed -i -- 's/sls-api-template-go/${repo_name}/g' ${functionpath}/main.go"
```

Comment:
User code shouldn't have the text - `sls-api-template-go` in the code at this point. This is template specific and should have been overridden when creating the service code from template during service creation. 

Can we move this line to service-onboarding-buid-pack, may be in the stage - `Update Service Template` https://github.com/tmobile/jazz/blob/sls_app/builds/service-onboarding-build-pack/Jenkinsfile#L191? You will need to add a condition before adding this line, something like -  `service_config['type'] == "sls-app" && service_config['runtime'].startsWith("go")`

Reply:
Moved on to service Onboarding build pack.